webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/domain/nodes/Properties.scala

```diff
@@ -233,11 +233,10 @@ object GenericProperty {
    */
   def parseValue(value: String): PureResult[ConfigValue] = {
     // find first char that is no in a commented line nor a space
-
     if(value == "") Right(ConfigValueFactory.fromAnyRef(""))
     else firstNonCommentChar(value) match {
-      case None => // here, either we return empty string, or the orginal one. I thing we should return empty string, since user can quote if he wants.case _: scala.None.type =>
-        Right(ConfigValueFactory.fromAnyRef(""))
+      case None => // here, we need to return the original string, user may want to use a comment (in bash for ex) as value
+        Right(ConfigValueFactory.fromAnyRef(value))
       case Some(c) if(c == '{') =>
```

Comment:
does that mean that a string variable cannot start by `{`?

Reply:
I tested and it says:
> if you intended '}' to be part of a key or string value, try enclosing the key or value in double quotes

which is not true as the quotes are then part of the value

Reply:
I think you're right. So if you really want  a string that starts with `{` (and is not json escapped, ie not actually `\"{` etc), you will need to build the json:
```
{ "whatever_key":"{ here your string" }
```

Reply:
Hum, ok this is a change from `6.0`

Reply:
https://issues.rudder.io/issues/17492